cEP-0010.md

```diff
@@ -139,7 +142,17 @@ Here is the list of aspects to be implemented along with their corresponding
 - `Root.Formatting.Quotation`: This aspect checks on the quotations being used
   in your files given `Root.Formatting.Quotation.preferred_quotation`.
 - `Root.Formatting.Quotation.preferred_quotation`: This taste represents the
-  user's preferred quotation.
+  preferred quotation marks for the codebase. It ensures that every string
+  contains the preferred quotes.
```

Comment:
This taste represents the preferred quotation marks for the codebase. It ensures that every string contains the selected style of quotes.